conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/secure.rs

```diff
@@ -38,9 +38,6 @@ fn can_validate_without_delegation() -> Result<()> {
     assert!(output.status.is_noerror());
     assert!(output.flags.authenticated_data);
 
-    let output = client.delv(resolver_addr, RecordType::SOA, &FQDN::ROOT, trust_anchor)?;
```

Comment:
this change is done in this PR because these assertions would have failed after the SERVFAIL change introduced in the next commit. turns out the missing test (PR for that coming up soon) described in #2274 uncovers a bug in the AD logic of the `recursor` authority

(interestingly, `delv` does *not* set the CD bit by default but also does not look if the AD bit is set in the response. it does abort when any response has the status SERVFAIL. those defaults probably let it inter-op with both security-aware, *non*-validating resolvers; and validating resolvers)

Reply:
Suggestion: maybe prefix commit messages that exclusively touch the conformance tests with `conformance: `? Would help me filter on review.